models/tridimensional/kleinstiver_csv_to_dict.py

```diff
@@ -0,0 +1,65 @@
+"""
+Convert CSV of Cas9 mutants from Klienstiver et al. (2015), Nature, doi:10.1038/nature14592 into dict format
+Included for reproducibility of data preparation
+"""
+
+from cas9_mutants import PI_domain, PI_sec_structure, aa_group
+import os
+import csv
+
```

Comment:
"Imports are always put at the top of the file, just after any module comments and docstrings, and before module globals and constants." - https://www.python.org/dev/peps/pep-0008/

So I think comments stay above, but will reorder the import statements.